frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -220,6 +220,16 @@ if Device:isAndroid() then
         end,
     }
 
+    -- camera key events
+    common_settings.android_camera_keys = {
```

Comment:
`android_camera_key` singular as there is only one (even if it kills the symetry with the one above :)